src/components/Menu/MuiFormControl/myMuiForm.tsx

```diff
@@ -0,0 +1,170 @@
+  FormControl,
+  SelectChangeEvent,
+  Typography,
+  SxProps,
+} from "@mui/material";
+import { colors } from "../../../theme";
+
+interface FormControlProps {
+  fullwidth?: boolean;
+  sx?: SxProps;
+  variant?: string; //outlined & standard
+  height?: string;
+  width?: string;
+  fontweight?: string;
+  padding?: string;
+  borderColor?: string;
+  fontWeight?: string;
+}
+
+interface SelectFieldProps extends FormControlProps {
+  id: string;
+  value: string | string[];
+  name?: string;
+  multiple?: boolean;
+  IconComponent?: React.ElementType;
+  labelId?: string;
+  variant?: "outlined" | "standard";
+  onChange?: (event: SelectChangeEvent) => void;
+  options: { value: string; label: string | React.ReactNode }[];
+  label?: string;
+  width?: string;
+  placeholder?: string;
+}
+
+export default function myMuiForm(props: Readonly<SelectFieldProps>) {
+  const { id, options } = props;
+
+  return (
+    <FormControl
+      fullWidth
+      sx={{
+        ".MuiInputBase-root": {
+          width: props.width ?? "100%",
+          height: props.height ?? "32px",
+          borderRadius: "5px",
+          background:
+            props.variant === "outlined"
+              ? colors.secondary.lightGrey
+              : colors.primary.blue,
+          border:
+            props.variant === "outlined"
+              ? `1px solid ${props.borderColor ?? colors.secondary.iconGray}`
+              : "none",
+        },
+        ".MuiInputBase-input": {
+          fontSize: "12px",
+          fontWeight: props.fontWeight ?? "700",
+          lineHeight: "normal",
+          textAlign: "start",
+          padding: props.variant === "outlined" ? "10px 20px" : "0px 8px",
+          color:
+            props.variant === "outlined"
+              ? colors.text.title
+              : colors.primary.white,
+        },
+        ".MuiOutlinedInput-notchedOutline": {
+          borderStyle: "none",
+        },
+        ".MuiSelect-icon": {
+          color: colors.primary.white,
+        },
+        ".MuiMenuItem-root": {
+          fontSize: "12px",
+          fontWeight: "700",
+        },
+        ".MuiSelect-paper": {
+          background: `colors.primary.green !important`,
+          padding: "100px",
+          borderRadius: "10px",
+          border: `1px solid ${colors.secondary.iconGray}`,
+        },
+      }}
+    >
+      {props.label ? (
+        <Typography
+          lineHeight={"normal"}
+          variant="body2"
+          fontSize={"12px"}
+          component={"label"}
+          htmlFor={props.id}
+          marginBottom={"5px"}
+        >
+          {props.label}
+        </Typography>
+      ) : null}
+      <Select
+        id={props.id}
+        value={props.value as string}
+        name={props.name}
+        displayEmpty
+        multiple={props.multiple}
+        IconComponent={props.IconComponent}
+        labelId={`${id}-label`}
+        onChange={props.onChange}
+        renderValue={(selected: string | string[]) => {
```

Comment:
declare this as a function at the top instead, and call the function here. Let's avoid inline function declarations